src/components/Header.jsx

```diff
@@ -7,22 +7,38 @@ import PageIndicatorContainer from "../containers/PageIndicatorContainer";
 class Header extends Component {
   render() {
     const currentPage = this.props.pageNum;
+    const summary = this.props.summary;
+
     const totalPages = "8";
-    return (
-      <div className="header-container">
-        <div>
-          <img className="logo" src={`/${logo}`} alt="Logo" />
+
+    if (summary) {
+      return (
+        <div className="header-container summary-header">
+          <div>
+            <img className="logo" src={`/${logo}`} alt="Logo" />
+          </div>
+          <h2 className="text header">Dataset Bias Assessment Tool</h2>
+          <h1 className="text title"> Summary of Results </h1>
+          {/* <br></br> */}
         </div>
-        <h2 className="text header">Dataset Bias Assessment Tool</h2>
-        <p className="text subheader">
-          Section {currentPage}/{totalPages}
-        </p>
-        <h1 className="text title">Progress Title</h1>
-        <div className="indicator-container">
-          <PageIndicatorContainer pageNum={currentPage} />
+      );
+    } else {
+      return (
+        <div className="header-container">
+          <div>
+            <img className="logo" src={`/${logo}`} alt="Logo" />
+          </div>
+          <h2 className="text header">Dataset Bias Assessment Tool</h2>
+          <p className="text subheader">
+            Section {currentPage+1}/{totalPages}
+          </p>
+          <h1 className="text title">Progress Title</h1>
+          <div className="indicator-container">
+            <PageIndicatorContainer pageNum={currentPage} />
+          </div>
```

Comment:
Oh this Header stuff is something I'll was going to take on. This seems like a good direction though, I can build off of it.